### PR TITLE
MRC-187 Accessibility Audit Fixes

### PR DESCRIPTION
**Pull request by adamjw3**

### CODE REVIEW

File: app/views/DistributionView.scala.html [hunk 1]
```diff
@@ -29,7 +29,17 @@
...
+                ErrorSummary(
+                    errorList = Seq(
+                        ErrorLink(
+                            href = Some("#value_0"),
+                            content = Text(messages("distribution.error.required"))
+                        )
+                    ),
+                    title = Text(messages("distribution.errorsummary")),
+                )
+            )
```
Comment (girishkamat) [inline app/views/DistributionView.scala.html:42]: Not sure why this is needed. Can we have a chat when you are available?

File: app/views/helpers/ResultsPageHelper.scala [hunk 2]
```diff
@@ -382,7 +384,8 @@ object ResultsPageHelper extends ViewHelper {
               TableRow(content = Text(PercentageUtils.format(s.details.taxRate)))
             )
           ).filter(_.nonEmpty),
-          caption = None,
+          caption = Some("Effective tax rate breakdown for your accounting period"),
```
Comment (girishkamat) [inline app/views/helpers/ResultsPageHelper.scala:387]: Do we need translations for this?

Comment (adamjw3) [reply app/views/helpers/ResultsPageHelper.scala]: Yeah, i had created the string var but missed it in a couple of places

File: app/views/DistributionView.scala.html (hunk 1 above)
Comment (adamjw3) [reply app/views/DistributionView.scala.html]: Updated and simplified this

**Review by girishkamat — APPROVED** [review]

**Pull request merged**